rakelib/tests.rake

```diff
@@ -67,32 +68,34 @@ begin
     namespace :linux do
       # Run RSpec tests that do not require WinRM
       desc ''
-      RSpec::Core::RakeTask.new(:fast) do |t|
-        t.rspec_opts = '--tag ~winrm --tag ~lxd_transport --tag ~windows_agents --tag ~puppetserver ' \
-                       '--tag ~puppetdb --tag ~omi --tag ~windows --tag ~kerberos --tag ~expensive ' \
-                       '--tag ~lxd_remote --tag ~podman'
+      RSpec::Core::RakeTask.new(:unit) do |t|
+        t.pattern = "spec/unit/**/*_spec.rb,spec/pal/**/*_spec.rb"
+        t.rspec_opts = '--tag ~winrm'
       end
 
       # Run RSpec tests that are slow or require slow to start containers for setup
       desc ''
-      RSpec::Core::RakeTask.new(:slow) do |t|
-        t.rspec_opts = '--tag puppetserver --tag puppetdb --tag expensive'
+      RSpec::Core::RakeTask.new(:integration) do |t|
+        t.pattern = "spec/integration/**/*_spec.rb,spec/bolt_server/**/*_spec.rb,spec/bolt_spec/**/*_spec.rb"
+        t.rspec_opts = '--tag ~winrm --tag ~lxd_transport --tag ~lxd_remote ' \
+                        '--tag ~podman --tag ~kerberos --tag ~omi --tag ~windows_agents'
       end
     end
 
     namespace :windows do
       # Run RSpec tests that do not require Puppet Agents on Windows
       desc ''
-      RSpec::Core::RakeTask.new(:agentless) do |t|
-        t.rspec_opts = '--tag ~ssh --tag ~docker --tag ~lxd_transport --tag ~bash --tag ~windows_agents ' \
-                       '--tag ~orchestrator --tag ~puppetserver --tag ~puppetdb --tag ~omi ' \
-                       '--tag ~kerberos --tag ~lxd_remote --tag ~podman'
+      RSpec::Core::RakeTask.new(:unit) do |t|
+        t.pattern = "spec/unit/**/*_spec.rb,spec/pal/**/*_spec.rb"
+        t.rspec_opts = '--tag ~bash --tag ~ssh --tag ~podman'
```

Comment:
```suggestion
        t.rspec_opts = '--tag ~bash --tag ~ssh'
```
It doesn't look like we have any podman tags in unit, only `winrm`, `bash`, and `ssh`.